pkg/operator/ceph/cluster/osd/config.go

```diff
@@ -91,6 +108,20 @@ func (c *Cluster) osdRunFlagTuningOnPVC(osdID int) error {
 	return nil
 }
 
+func (c *Cluster) osdRunFlagFastTuningOnPVC(osdID int) error {
+	who := fmt.Sprintf("osd.%d", osdID)
+	monStore := opconfig.GetMonStore(c.context, c.clusterInfo)
+
+	for flag, val := range defaultTuneFastSettings {
+		err := monStore.Set(who, flag, val)
+		if err != nil {
+			return errors.Wrapf(err, "failed to set %q to %q on %q", flag, val, who)
```

Comment:
Let's just log the error instead of failing the reconcile. This will also simplify the goroutine.

Reply:
Don't need now, we moved cli. Thanks